deps/wazuh_testing/wazuh_testing/end_to_end/vulnerability_detector.py

```diff
@@ -305,14 +305,19 @@ def get_vulnerabilities_from_states(vulnerabilities_states: List) -> List:
         try:
             vulnerability = Vulnerability(
                 cve=state_vulnerability['_source']['vulnerability']['id'],
-                package_name=state_vulnerability['_source']['package']['name'],
-                package_version=state_vulnerability['_source']['package']['version'],
-                type=state_vulnerability['_source']['pacakge']['type'] if 'type' in state_vulnerability['_source']['vulnerability'] else None,
-                architecture=state_vulnerability['_source']['package']['architecture'] if 'architecture' in state_vulnerability['_source']['vulnerability'] else None
+                package_name=(state_vulnerability['_source']['package']['name']
+                              if 'package' in state_vulnerability['_source'] else None),
+                package_version=(state_vulnerability['_source']['package']['version']
+                                 if 'package' in state_vulnerability['_source'] else None),
+                type=(state_vulnerability['_source']['package']['type']
+                      if 'type' in state_vulnerability['_source']['vulnerability'] else None),
+                architecture=(state_vulnerability['_source']['package']['architecture']
+                              if 'architecture' in state_vulnerability['_source']['vulnerability'] else None)
```

Comment:
Sorry, I didn't see this, but the `if` check appears to be wrong, can you check it? I think it should be

> type=(state_vulnerability['_source']['package']['type']
>                       if 'type' in state_vulnerability['_source']['`package`'] else None),
> architecture=(state_vulnerability['_source']['package']['architecture']
>                       if 'architecture' in state_vulnerability['_source']['`package`'] else None)

Reply:
Done in fbdadb2